### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by pablo-clueless**
- change ur life section into a full-screen
- the color of the paragraph under change ur life header
- course fee header capitalized
- text color of the enroll now button in the course fee section fixed

### CODE REVIEW

**Comment by ELDEVODE** [comment]
There are conflicts boss

**Comment by pablo-clueless** [comment]
> There are conflicts boss

Fixed sir., thanks

**Review by ELDEVODE — APPROVED** [review]

File: src/Section/AcaSecTwo.jsx [hunk 1]
```diff
@@ -31,6 +33,7 @@ function AcaSecTwo() {
           sx={{
             fontSize: '20px'
           }}
+          color="text.secondary"
```
Comment (taiwoAkindele) [inline src/Section/AcaSecTwo.jsx:36]: Please check this. The color on the design is much faded out than this.

**Review by taiwoAkindele — APPROVED** [review]

File: src/Section/AcaSecTwo.jsx (hunk 1 above)
Comment (pablo-clueless) [reply src/Section/AcaSecTwo.jsx]: I have copied the exact color code into it.

**Pull request merged**